system_tests/bigquery.py

```diff
@@ -29,6 +29,11 @@
 
 DATASET_NAME = 'system_tests' + unique_resource_id()
 
+# We need to wait to stay within the rate limits.
+# The alternative outcome is a 403 Forbidden response from upstream.
+# See: https://cloud.google.com/bigquery/quota-policy
```

Comment:
Thanks for the note, moved the error predicate discussion to the main thread since GitHub "outdated" this diff